packages/core/content-releases/admin/src/services/release.ts

```diff
@@ -203,9 +199,7 @@ const releaseApi = createApi({
             data: body,
           };
         },
-        invalidatesTags: (result, error, arg) => [
-          { type: 'ReleaseAction', id: arg.params.actionId },
-        ],
+        invalidatesTags: () => [{ type: 'ReleaseAction', id: 'LIST' }],
```

Comment:
not familiar with this, where does id LIST come from?

Reply:
It's recommended in the redux-toolkit docs, it's pretty neat, you can then invalidate the whole list when you create a new one and invalidate the cache of an individual list item when it's updated or deleted.